deploy/helm/sumologic/values.yaml

```diff
@@ -4145,6 +4145,22 @@ metadata:
           sending_queue:
             enabled: true
             persistent_storage_enabled: '{{ .Values.metadata.persistence.enabled }}'
+        ## This is the same like sumologic/systemd with additional `flatten_body` parameter
+        sumologic/systemd_flatten:
+          log_format: json
+          json_logs:
+            add_timestamp: true
+            timestamp_key: timestamp
+            flatten_body: true
```

Comment:
we can consider it for fluentd pipeline as well, but we need further investigation for that

Reply:
Question is if the systemd log can be nested json

Reply:
https://stackoverflow.com/questions/45604563/injecting-structured-json-logs-into-journald

```
journald doesn't accept arbitary JSON. Just Key/Value pairs. So it's not possible to send nested data structures
```